mailbag/controller.py

```diff
@@ -90,6 +95,16 @@ def generate_mailbag(self):
                      message.To, message.Cc,message.Bcc, message.Subject, message.Content_Type])
             csv_portion_count += 1
 
+            #creating text file filr and csv if error is present
+            if len(message.Error) > 0:
+                list=[message.Mailbag_Message_ID,message.Original_Filename,message.Error]
+                error_csv.append(list)
+                filename=os.path.join(error_dir,str(message.Mailbag_Message_ID)+'.txt')
+                c=traceback.extract_stack()
```

Comment:
I don't think `traceback.extract_stack()` is what we're looking for. You shouldn't need `traceback`. The stack trace is caught as `e` where we're `log.error(e)`, so we have to get that back to the controller somehow.